test-data/src/test/resources/org/openmrs/module/fhir2/narrative/expected/allergyintolerance-expected.html

```diff
@@ -1 +1 @@
-<div xmlns="http://www.w3.org/1999/xhtml"><table class="hapiPropertyTable"><tbody><tr><td>Id:</td><td>AllergyIntolerance/example</td></tr><tr><td>Identifier:</td><td><div>49476534</div></td></tr><tr><td>Clinical Status:</td><td> Active </td></tr><tr><td>Verification Status:</td><td> Confirmed </td></tr><tr><td>Type:</td><td>Allergy</td></tr><tr><td>Category:</td><td> food </td></tr><tr><td>Criticality:</td><td>HIGH</td></tr><tr><td>Code:</td><td> Cashew nuts </td></tr><tr><td>Patient:</td><td><a href="http://localhost:8080/openmrs/ws/fhir2/R4/Patient/example">Patient/example</a></td></tr><tr><td>Onset:</td><td> 2004 </td></tr><tr><td>Recorded Date:</td><td>09/10/2014</td></tr><tr><td>Recorder:</td><td><a href="http://localhost:8080/openmrs/ws/fhir2/R4/Practitioner/example">Practitioner/example</a></td></tr><tr><td>Asserter:</td><td><a href="http://localhost:8080/openmrs/ws/fhir2/R4/Patient/example">Patient/example</a></td></tr><tr><td>Last Occurrence:</td><td>01/06/2012</td></tr><tr><td>Note:</td><td><div>The criticality is high becasue of the observed anaphylactic reaction when challenged with cashew extract.</div></td></tr><tr><td>Reaction:</td><td/></tr><tr><td>Substance:</td><td> cashew nut allergenic extract Injectable Product </td></tr><tr><td>Manifestation:</td><td><div> Anaphylactic reaction </div></td></tr><tr><td>Description:</td><td>Challenge Protocol. Severe reaction to subcutaneous cashew extract. Epinephrine administered</td></tr><tr><td>Onset:</td><td>12/06/2012</td></tr><tr><td>Severity:</td><td>SEVERE</td></tr><tr><td>Exposure Route:</td><td> Subcutaneous route </td></tr><tr><td>Reaction:</td><td/></tr><tr><td>Manifestation:</td><td><div> Urticaria </div></td></tr><tr><td>Onset:</td><td>01/01/2004</td></tr><tr><td>Severity:</td><td>MODERATE</td></tr><tr><td>Note:</td><td>The patient reports that the onset of urticaria was within 15 minutes of eating cashews.</td></tr></tbody></table></div>
```

Comment:
Shouldn't we expect that the output is the same?

Reply:
Actually it's the `:` which is missing after the property names.

Reply:
Have added the missing `:`.